tests/wpt/web-platform-tests/html/semantics/forms/attributes-common-to-form-controls/dirname-ltr.html

```diff
@@ -20,10 +20,12 @@
   }
 
   var t = async_test("submit element directionality");
-  document.querySelector("input").value="foobar";
-  document.querySelector("button").click();
+  setTimeout(function() {
+    document.querySelector("input").value="foobar";
+    document.querySelector("button").click();
 
-  document.querySelector("iframe").onload = t.step_func_done(function() {
-    assert_equals(getParameterByName("comment.dir"), "ltr");
-  });
+    document.querySelector("iframe").onload = t.step_func_done(function() {
+      assert_equals(getParameterByName("comment.dir"), "ltr");
+    });
+  }, 3000);
```

Comment:
If this is because you were getting onload events for about:blank, a probably better way to do it is to check during the onload whether it's about:blank or not and only call t_step_func_done when it's not about:blank. Making tests wait for amounts of time before they do things can add up, and it's better to avoid it when possible.

Reply:
example: https://github.com/servo/servo/pull/25220/files#diff-2e857054edbb4a084f0deb6e79b7dac9

Reply:
Ah, thanks for the example!